packages/moonstone/ContextualPopupDecorator/ContextualPopupDecorator.js

```diff
@@ -546,16 +546,9 @@ const Decorator = hoc(defaultConfig, (config, Wrapped) => {
 		}
 
 		// handle key event from contextual popup and closes the popup
-		handleContainerKeyDown = (ev) => {
-			// Note: Container will be only rendered if `open`ed, therefore no need to check for `open`
-			const direction = getDirection(ev.keyCode);
-
-			if (!direction) return;
-
-			this.handleDirectionalKey(ev);
-
+		handleContainerBlur = (ev) => {
 			// if focus moves outside the popup's container, issue the `onClose` event
-			if (Spotlight.move(direction) && !this.containerNode.contains(Spotlight.getCurrent())) {
+			if (!this.containerNode.contains(Spotlight.getCurrent())) {
```

Comment:
I'm not sure that `Spotlight.getCurrent()` is valid here since this handler is called on `blur`.

Reply:
As I understand, when `Spotlight.getCurrent` is called, the returned value will be always `null`. So I think that we actually don't need the latter condition here. That was my question.

Reply:
`Spotlight.getCurrent()` will return next target element instead `null`. because _current element will set on navbeforefocus.